<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>SpectraViewer</class>
    <widget class="QMainWindow" name="SpectraViewer">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1148</width>
                <height>690</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="2" rowspan="4">
                    <widget class="QPushButton" name="pushButton_2">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>18</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="text">
                            <string>Â«</string>
                        </property>
                    </widget>
                </item>
                <item row="2" column="0" colspan="2">
                    <widget class="QLabel" name="lab_sf">
                        <property name="text">
                            <string>sf</string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="0" colspan="2">
                    <widget class="QTableWidget" name="tableMassCal">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <property name="selectionMode">
                            <enum>QAbstractItemView::SingleSelection</enum>
                        </property>
                        <property name="selectionBehavior">
                            <enum>QAbstractItemView::SelectRows</enum>
                        </property>
                        <property name="columnCount">
                            <number>4</number>
                        </property>
                        <attribute name="horizontalHeaderDefaultSectionSize">
                            <number>64</number>
                        </attribute>
                        <column>
                            <property name="text">
                                <string>Element</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Mass</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Time</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Delta [u]</string>
                            </property>
                        </column>
                    </widget>
                </item>
                <item row="3" column="0" colspan="2">
                    <widget class="QLabel" name="lab_k0">
                        <property name="text">
                            <string>k0</string>
                        </property>
                    </widget>
                </item>
                <item row="4" column="0" colspan="2">
                    <widget class="QPushButton" name="pushButton">
                        <property name="text">
                            <string>Delete Element</string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="3" rowspan="4">
                    <widget class="MplWidget" name="mpl" native="true">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <property name="styleSheet">
                            <string notr="true"/>
                        </property>
                    </widget>
                </item>
                <item row="0" column="3">
                    <widget class="QLabel" name="lab_m0">
                        <property name="text">
                            <string>Center Mass:</string>
                        </property>
                    </widget>
                </item>
                <item row="0" column="0" colspan="2">
                    <widget class="QCheckBox" name="show_mass">
                        <property name="enabled">
                            <bool>false</bool>
                        </property>
                        <property name="text">
                            <string>Show masses</string>
                        </property>
                        <property name="checkable">
                            <bool>true</bool>
                        </property>
                        <property name="checked">
                            <bool>true</bool>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1148</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <customwidgets>
        <customwidget>
            <class>MplWidget</class>
            <extends>QWidget</extends>
            <header>.mplwidget</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
